client/logic/game/maps/map_file.gd

```diff
@@ -1,7 +1,7 @@
 class_name MapFile
 extends Node
 
-const MAP_VERSION: String = "1.0.0"
+const MAP_VERSION: String = "1.0.1"
```

Comment:
this usually should go with a code of compatibility to load the previous version and avoid problems.